climada/hazard/tc_tracks.py

```diff
@@ -411,10 +499,14 @@ def read_ibtracs_netcdf(self, provider=None, storm_id=None,
                 last_perc = perc
             track_ds = ibtracs_ds.sel(storm=i_track, date_time=t_msk)
             st_penv = xr.apply_ufunc(basin_fun, track_ds.basin, vectorize=True)
-            track_ds['time'][:1] = track_ds.time[:1].dt.floor('H')
+
+            # set time_step in hours
+            track_ds.time.values[:1] = track_ds.time[:1].dt.floor('H')
```

Comment:
I'm aware that this line already existed and you did not add anything here, but I am not sure I understand why you need to round the first time step to the lower hour. Do you?

Reply:
I vaguely remember that this wasn't related to setting the time steps, but there was some kind of problem with the interpolation in `equal_timestep` when the first time stamp wasn't rounded to 1 hour. I agree that this requires clarification, but not necessarily in the context of this PR. See here: https://github.com/CLIMADA-project/climada_python/issues/134